### PR TITLE
Add docs for RDP NLA

### PR DESCRIPTION
**Pull request by zmb3**
Replaces #45099
Updates #8546

### CODE REVIEW

File: docs/pages/includes/config-reference/desktop-config.yaml [hunk 1]
```diff
@@ -70,6 +70,17 @@ windows_desktop_service:
...
+  # (optional) Configures the address of the Kerberos Key Distribution Center,
+  # which is used to support RDP Network Level Authentication (NLA).
+  # If empty, the LDAP address will be used instead.
+  #
+  # example: kdc.example.com:88.
+  # The port is optional and defaults to port 88 if unspecified.
+  #
+  # Note: NLA is only supported in Active Directory environments - this field has
+  # no effect when connecting to desktops as local Windows users.
+  kdc_addr: '$KDC_SERVER_ADDRESS'
```
Comment (zmb3) [inline docs/pages/includes/config-reference/desktop-config.yaml:82]: This should read `kdc_address`, not addr. Will fix.

**Review by ptgott — APPROVED** [review]

File: docs/pages/enroll-resources/desktop-access/active-directory.mdx [hunk 2]
```diff
@@ -737,9 +743,64 @@ domain, you would do the following:
...
+In order to complete a successful authentication flow, Teleport needs to know
+the hostname of the target Windows server. If you are using Teleport's
+LDAP-based discovery no action is necessary - Teleport will default to using the
+`teleport.dev/computer_name` label which is automaticaly populated. If this
+label is not present then Teleport will use the desktop's `addr` field.
+
+If you register your Windows hosts via configuration file or via Teleport's API
+and you want to enable NLA, then you should either:
+
+- manually set the `teleport.dev/dns_host_name` label to the proper hostname
```
Comment (probakowski) [inline docs/pages/enroll-resources/desktop-access/active-directory.mdx:798]:
```suggestion
- manually set the `teleport.dev/computer_name` label to the proper hostname
```

**Review by probakowski — CHANGES_REQUESTED** [review]

**Review by mmcallister — APPROVED** [review]

**Review by probakowski — APPROVED** [review]

File: docs/pages/enroll-resources/desktop-access/active-directory.mdx [hunk 3]
```diff
@@ -737,9 +743,64 @@ domain, you would do the following:
...
+only supported when connecting to hosts that are part of an Active Directory
+domain. Teleport will not perform NLA when connecting to hosts as a local
+Windows user.
+
+### Computer Name
+
+In order to complete a successful authentication flow, Teleport needs to know
+the hostname of the target Windows server. If you are using Teleport's
+LDAP-based discovery no action is necessary - Teleport will default to using the
+`teleport.dev/computer_name` label which is automaticaly populated. If this
```
Comment (mmcallister) [inline docs/pages/enroll-resources/desktop-access/active-directory.mdx:792]:
```suggestion
`teleport.dev/computer_name` label which is automatically populated. If this
```

File: docs/pages/enroll-resources/desktop-access/troubleshooting.mdx [hunk 4]
```diff
@@ -393,3 +393,25 @@ to force the policy update:
...
+This is an error with NLA. To connect to a Windows host using NLA, Teleport
+must specify the computer name as it exists in Active Directory. You will see
+this error if you have not specified the correct computer name, or if you are
+connecting by IP address instead of hostname.
+
+**Solution:** correct the server's computer name
+
+To fix this error, ensure that either:
+
+- the server's computer name is specified in the `telepot.dev/computer_name` label
```
Comment (mmcallister) [inline docs/pages/enroll-resources/desktop-access/troubleshooting.mdx:414]:
```suggestion
- the server's computer name is specified in the `teleport.dev/computer_name` label
```

**Review by webvictim — APPROVED** [review]

File: docs/pages/enroll-resources/desktop-access/troubleshooting.mdx [hunk 5]
```diff
@@ -393,3 +393,25 @@ to force the policy update:
 ```code
 $ gpupdate.exe /force
 ```
+
+### CredSSP: server not found in Kerberos database
```
Comment (webvictim) [inline docs/pages/enroll-resources/desktop-access/troubleshooting.mdx:397]: Do you think it's also worth adding `CredSSP enhanced RDP security required by server` as a header here, with instructions about setting the environment variable?

**Pull request merged**